change/@fluentui-react-charting-e365c5ce-592e-422f-9329-1bee461a0dc3.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "Fix for bug 9136 and 7997",
```

Comment:
Can you add description about the issue here and not only the bug id

Reply:
Done